x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/Spans/SpanDetails/index.js

```diff
@@ -77,6 +78,12 @@ const LegendIndicator = styled(Indicator)`
   display: inline-block;
 `;
 
+const SectionHeader = styled.div`
+  margin-top: ${px(unit)};
+  font-size: ${fontSize};
+  color: ${colors.gray1};
+`;
+
```

Comment:
Beautiful! 
We already have [a couple of headers](https://github.com/sqren/kibana/blob/3988e3639f7d354dc9b5578daf56b4d93fc00f53/x-pack/plugins/apm/public/components/shared/UIComponents.js#L11-L41). Maybe we could go over them at some point.

Reply:
@sqren Made sense to re-use those, although added a new smaller size to match the other style I created.